scripts/linters/js_ts_linter.py

```diff
@@ -210,21 +209,20 @@ def _get_expressions_from_parsed_script(self):
 
         return parsed_expressions_in_files
 
-    def _compile_ts_file(self, filepath, dir_path):
-        """Compiles a typescript file and returns the path for compiled
-        js file.
-        """
-        cmd = (
-            './node_modules/typescript/bin/tsc -outDir %s -allowJS %s '
-            '-lib %s -noImplicitUseStrict %s -skipLibCheck '
-            '%s -target %s -typeRoots %s %s typings/*') % (
-                dir_path, 'true', 'es2017,dom', 'true',
-                'true', 'es5', './node_modules/@types', filepath)
-        subprocess.call(cmd, shell=True, stdout=subprocess.PIPE)
+    def _get_compiled_ts_filepath(self, filepath):
+        """Returns the path for compiled ts file."""
```

Comment:
Missing Args, Returns part of docstring.

Reply:
Done